docs/userguide/transfer_learning.md

```diff
@@ -1,4 +1,41 @@
 # Transfer Learning
 
-This page will soon contain information about transfer learning.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers the possibility to mix data from multiple campaigns in order to accelerate
+optimization. 
+Using data from multiple campaigns is currently supported when using a Gaussian Process
+Surrogate model and is implemented by the [`TaskParameter`](baybe.parameters.categorical.TaskParameter).
+
+```{admonition} Terminology
+:class: note
+The term "Transfer Learning" is used in a lot of different ways.
+The act of combining the data of several contexts can be done via multiple models,
+shared architectures, special kernels in a single model and whatnot.
+We do not necessarily want to limit to any of these methods, even though BayBE currently
+offers only a single one.
+```
+
+## The role of ``TaskParameter``
+
+The ``TaskParameter`` is used to "mark" the context of an individual experiment. The
+set of all possible contexts is provided upon the initialization of a ``TaskParameter``
+by providing them as ``values``.
+
+```python
+from baybe.parameters import TaskParameter
+
+TaskParameter(name="Month", values=["Nov23", "Dec23", "Jan24"])
+```
+
+If not specified further, a ``campaign`` using the ``TaskParameter`` as specified above
```

Comment:
are you referring to the Capaign class? or a variable? In any case, none of these are used and I would write campaign without backticks

Reply:
Removed the backticks.